### ISSUE TITLE
0.3 Cashes when starting on x86 Android 5.0

### ISSUE THREAD

**Issue by poVoq** [UnevenSoftware/LeafPic#163]
Just updated via F-droid to the latest version 0.3 and now it refuses to run (up to 0.27 it was fine except for the OpenCamera bug). All I get is the white background with the leafpic logo and then a "Unfortunatly leafpic has stopped" message.

This is on a stock Android 5.0 x86 Zenfone2.

**Comment by iasgair** [comment]
v0.3 crashes for me as well. 
My screen is grey with the leafpic logo (because I'm using the dark theme?) and then the same message. 
Android 5.1, Fairphone 2, everything google deactivated

**Comment by dnldsht** [comment]
Can you provide me the log?

**Comment by petrblaho** [comment]
The same for me on OmniROM (6.0.1-20160419-mako-NIGHTLY) on Nexus 4, minimal google apps.

Edit: better have the logs in gist - https://gist.github.com/petrblaho/dc513ea0ca4a32df38aaa3335c73c9e1

**Comment by Valeria-AT** [comment]
0.3 is crashing too on my system, but with no message.

Android 5.1.1
No Google Play Services
MIUI 7.2.7.0
Xiaomi Redmi 3

**Comment by gerroon** [comment]
This version crashes on Android 4.4.4 Arm device as well. It complains about not finding setstatusbarcolor in the log, might be related

https://paste.debian.net/hidden/bd2c0dbc/

**Comment by dnldsht** [comment]
can you guys test this [apk](https://drive.google.com/file/d/0B4G2W-xokbBreTZGNEtBaTM4Q28/view?usp=sharing) and tell me if it crashes during the launch?

**Comment by Valeria-AT** [comment]
0.2.7 updating to app-debug: failed install
0.2.7 uninstall and app-debug install: successful install and successful launching

**Comment by dnldsht** [comment]
is normal that the install failed, are you able to show all your images?

**Comment by Valeria-AT** [comment]
Yes all media folders and all pictures are displayed and i can open them, but it's very slow.

**Comment by dnldsht** [comment]
it takes a lot to get media or to display them?

**Comment by Valeria-AT** [comment]
Switching folders (and going back to the main menu) takes long and scrolling through folders is laagy.

**Comment by dnldsht** [comment]
I suggest to disable auto update media in the settings and it will take a while to get images in the cache

**Comment by Valeria-AT** [comment]
Yes this improves the speed and switching to the main menu is instant instead of a few seconds^^
P.S.: My biggest folder has 491 pictures in it.

**Comment by iasgair** [comment]
Worked for me as well. It wasn't too slow either, one folder of about 450 pictures. Still, with auto update disabled, it is a bit quicker.

Thanks for the quick fix!

**Comment by poVoq** [comment]
Can confirm, this debug version works fine.

**Comment by gerroon** [comment]
dbg version works

**Comment by petrblaho** [comment]
Apk linked here works for me...

**Comment by iasgair** [comment]
Just for reference:
I updated from v0.27 to v0.3.1 yesterday using fdroid. When starting LeafPic, it again crashed right after the splash screen was displayed. Funnily, when evoking it from within OpenCamera (which now works a treat), everything was fine.
I then uninstalled LeafPic\* and installed v0.3.1 from fdroid and now everything works fine.

*if it matters: I uninstalled the app using android, not from within fdroid. I did install the newest version from within fdroid afterwards, though.
